src/test/ui/impl-trait/recursive-impl-trait-type-direct.stderr

```diff
@@ -1,10 +1,11 @@
-error[E0720]: opaque type expands to a recursive type
+error[E0720]: cannot resolve opaque return type `impl Sized` to a concrete type
   --> $DIR/recursive-impl-trait-type-direct.rs:5:14
    |
 LL | fn test() -> impl Sized {
-   |              ^^^^^^^^^^ expands to a recursive type
-   |
-   = note: type resolves to itself
+   |              ^^^^^^^^^^ cannot resolve to a concrete type
```

Comment:
Hmm, "concrete" here doesn't feel that specific. Maybe keep the "expands to a recursive type" label